ispdb/ptd.net

```diff
@@ -1,46 +1,60 @@
-      <username>%EMAILLOCALPART%</username>
-      <authentication>password-cleartext</authentication>
-    </incomingServer>
-    <incomingServer type="pop3">
-      <hostname>promail.ptd.net</hostname>
-      <port>110</port>
-      <socketType>STARTTLS</socketType>
-      <username>%EMAILLOCALPART%</username>
-      <authentication>password-cleartext</authentication>
-    </incomingServer>
-    <outgoingServer type="smtp">
-      <hostname>promail.ptd.net</hostname>
-      <port>587</port>
-      <socketType>STARTTLS</socketType>
-      <username>%EMAILLOCALPART%</username>
-      <authentication>password-cleartext</authentication>
-    </outgoingServer>
-    <documentation url="http://www.ptd.net/help-desk/email-communication/how-setup-your-email">
-      <descr lang="en">How to set up email</descr>
-    </documentation>
-  </emailProvider>
+    <emailProvider id="ptd.net">
+        <domain>ptd.net</domain>
+        <displayName>PenTeleData</displayName>
+        <displayShortName>PenTeleData</displayShortName>
+        <incomingServer type="imap">
+            <hostname>promail.ptd.net</hostname>
+            <port>993</port>
+            <socketType>SSL</socketType>
+            <username>%EMAILLOCALPART%</username>
+            <authentication>password-cleartext</authentication>
+        </incomingServer>
+        <incomingServer type="imap">
+            <hostname>promail.ptd.net</hostname>
+            <port>143</port>
+            <socketType>STARTTLS</socketType>
+            <username>%EMAILLOCALPART%</username>
+            <authentication>password-cleartext</authentication>
+        </incomingServer>
+        <incomingServer type="pop3">
+            <hostname>promail.ptd.net</hostname>
+            <port>995</port>
+            <socketType>SSL</socketType>
+            <username>%EMAILLOCALPART%</username>
+            <authentication>password-cleartext</authentication>
+        </incomingServer>
+        <incomingServer type="pop3">
+            <hostname>promail.ptd.net</hostname>
+            <port>110</port>
+            <socketType>STARTTLS</socketType>
+            <username>%EMAILLOCALPART%</username>
+            <authentication>password-cleartext</authentication>
+        </incomingServer>
+        <outgoingServer type="smtp">
+            <hostname>promail.ptd.net</hostname>
+            <port>465</port>
+            <socketType>SSL</socketType>
+            <username>%EMAILLOCALPART%</username>
+            <authentication>password-cleartext</authentication>
+        </outgoingServer>
+        <outgoingServer type="smtp">
+            <hostname>promail.ptd.net</hostname>
+            <port>587</port>
+            <socketType>STARTTLS</socketType>
+            <username>%EMAILLOCALPART%</username>
+            <authentication>password-cleartext</authentication>
+        </outgoingServer>
+        <outgoingServer type="smtp">
+            <hostname>promail.ptd.net</hostname>
+            <port>25</port>
```

Comment:
An email client should never have to use port 25 for submission. They do list this in their documentation. But I wouldn't include it here.

Reply:
@cketti - Thank you for the feedback. I pushed a commit that removes port 25.